webrtc.html

```diff
@@ -4166,7 +4166,7 @@ <h2 id="sec.stats-model">Statistics Model</h2>
 
       <dl class="idl" title="partial interface RTCPeerConnection">
         <dt>Promise&lt;RTCStatsReport&gt; getStats(
-          optional MediaStreamTrack? selector)</dt>
+          optional MediaStreamTrack? selector = null)</dt>
```

Comment:
Sure, but I'm not making my point well: What if the new getStats did take more arguments? Would we go with `null` or `undefined`? I think `null`.

Furthermore, I think `null` makes sense semantically here. E.g. in this case:

    var selector = findSelectorPreference();
    return getStats(selector);

Here `selector` can be `null`, which explicitly means "select all", which is different from `undefined` which might mean "no preference". Lets not force people to do this:

    return getStats((selector === null? undefined : selector);